File: 07_funcoes/ex03_maximo_tres.c
/*
Exercicio 03
Tema: 07_funcoes
*/

#include <stdio.h>

int maior(int * arr){
    int maior = arr[0];

    for(int i = 1; i < 3; i++){
        if (arr[i] > maior){
            maior = arr[i];
        }
    }
    return maior;
}

int main() {

    int arr[3];
    for(int i = 0; i < 3; i++){
        scanf("%d", &arr[i]);
    }

    int resultado = maior(arr);

    printf("%d", resultado);

    return 0;
}

File: 07_funcoes/ex03_maximo_tres.py
"""
Exercicio 03
Tema: 07_funcoes
"""

def maior(arr):
    maior = arr[0]

    for i in range(1, 3):
        if arr[i] > maior:
            maior = arr[i]
    return maior

arr = [0]*3

for i in range(3):
    arr[i] = int(input())

print(maior(arr))
